### DISCUSSION TITLE
Add option to not restart/loop music on song select

### DISCUSSION THREAD

**Discussion by SonoSooS** [ppy/osu#21382] [Ideas]
This has been a pet peeve of mine since forever, that after completing the beatmap, the music is still playing/looping, and I have to manually pause it by pressing F5 then F3. This is even worse in the proprietary client, where I have to fully go back to the main logo screen to be able to pause the music at all.

After completing a difficult beatmap, I'd prefer to wind down and take a breath for a few tens of seconds, but instead the same few seconds of loud beatmap keeps blaring into my ears.

Although it doesn't have to be only silenced after completing a beatmap. It would be great if it was possible to set it to "loop once", so on those beatmaps where the preview is at the end, it doesn't keep repeating the same short period over and over again, or if it was possible to just completely disable beatmap previews at all, as I recognize them from the cover anyways.

**Comment by peppy** [comment]
Verging on the side of "too much customisation" in my eyes.

**Reply by SonoSooS** [reply]
I thought about it some more, and it could be minimized into a new option: **music** volume slider outside of gameplay.

Since there are now mods to alter the in-game volume of the music, there is no longer a need for me to touch the volume slider ever again, as I play at fixed music volume, unless I mute it completely for some challanges.
This setting would allow me to completely mute the music in the menus, or if it didn't annoy me, then it could bring the volume UP again, so I could actually hear the preview of the beatmap, and not just hear some faint undistinguishable noise coming from the headphones.
